.github/workflows/greet.yml

```
@@ -0,0 +1,16 @@
+name: Welcome first time contributors
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    branches:
+      - main
```

```suggestion
      - master
```
?  Also, I _think_ that `[skip ci]` won't trigger CI :wink:

---

I am not sure what you mean here :/

---

If you have `[skip ci]` in the git commit message, this job won't trigger CI (which is useless and wasteful, since this PR doesn't touch julia code)